content/recipes/interoperability/bioactivity-profile.md

```diff
@@ -0,0 +1,200 @@
+# Bioactivity data profile
+
+[TOC]
+## Main objective
+This recipe shows how to prepare bioactivity data to meet the ChEMBL submission criteria, focusing on data formats, structures, and vocabularies. 
+
+## Graphical overview of the Recipe FAIRification Objectives
+![](https://i.imgur.com/qNuKXQ4.png)
+
+Bioactivity data, together with chemical data and omics data, can be used to search for new hits. FAIR bioactivity data guides the improvements of pharmacokinetic properties of compounds and the identification of drug targets. Early-stage bioactivity dataset includes compound molecular structure, molecular production details, assay data and pharmacokinetic study information.
+
+The [Minimum information about a bioactive entity (MIABE)](https://www.nature.com/articles/nrd3503) standards recommend attributes, formats and vocabularies for the reuse of such datasets. Public bioactivity data archives, such as [ChEMBL](https://www.ebi.ac.uk/chembl/), [PubChem](https://pubchem.ncbi.nlm.nih.gov/), and [ECBD](https://ecbd.eu/) also have their own requirements for data submission.
+
+### Formats and data models
```

Comment:
This table needs review has the values in the cells do not match the expected type defined by the field headers